README.md

```diff
@@ -24,12 +24,12 @@ Or install it yourself as:
 
 Decrypted secrets from `project/config/secrets.json` (or `project/config/secrets.{current_rails_environment}.json` if that doesn't exist) will be accessible via `Rails.application.secrets`. For example:
```

Comment:
Was this section and the line in the section above
>Automatically injects [`ejson`](https://github.com/Shopify/ejson) decrypted secrets into your `Rails.application.secrets`.
purposely kept as `Rails.application.secrets` (instead of `Rails.application.credentials`) until most apps are migrated off using Rails secrets?

Reply:
We could say "secrets and credentials".